students/students1.list

```diff
@@ -1,4 +1 @@
-Student name,Student Email
-Serhii Zhuhan
-Oleg Mastriukov
-
+Anna Blumenkrantz,sewingrus@gmail.com
```

Comment:
This file has conflicts. Please resolve them. For theory about conflicts please refer to [merge-conflicts.md](https://github.com/wedoforce/sf-dev-101-practice/blob/lessons/git-101/merge-confilcts.md)

Reply:
at the time of merge the branch was already up to date.